### DISCUSSION TITLE
debuff highlighting help

### DISCUSSION THREAD

**Discussion by spamwax** [rgd87/Aptechka#64] [Q&A]
During the council of blood fight, I was trying to show glowing pixel border for "Drain Essence" (327773) but couldn't get it to work.
This is how I set it up and it never showed up during all pulls. At first I thought I may have missed noticing it, but then it's a 6 sec debuff that's hard to miss if the border is glowing. 

<img width="339" alt="Wow_2020-12-22_0089" src="https://user-images.githubusercontent.com/1251233/102857907-50696680-43f7-11eb-9e08-ea46caeac04e.png">

**Comment by d87** [comment]
Looks like the spell id is wrong. I was adding them from wowhead before raid release, some spells consist from multiple effects with different ids that aren't always easy to find in the database or have several different versions of them. You can run ```/apt listauras target``` to get the correct id in-game.
And btw i don't raid so it's kinda hard for me to test raid highlights myself

**Comment by d87** [comment]
Judging from warcraftlogs data it's probably 346651. Or maybe not, channeling spells are weird

**Reply by spamwax** [reply]
ok, thanks. I am gonna add 3 different highlights with 346651, 327773 and 346654 and see if they work
